magento/magento2
# magento/magento2#25540: Products are not displaying infront end after updating product via importing CSV.

### Description (*)


### Fixed Issues (if relevant)

1. magento/magento2#25540: Products are not displaying infront end after updating product via importing CSV.

### Manual testing scenarios (*)

1. Create Simple product.
2. Create Bundle product, assign Simple to it.
3. Open Storefront, make sure Bundle product is visible and in stock.
4. Go to System -> Data Transfer -> Export and export products.
5. Go to System -> Data Transfer -> Import and import products.
6. Do reindex and flush cache.
7. Open Storefront, make sure Bundle product is visible and in stock.

### Questions or comments


### Contribution checklist (*)
